src/network/networkd-link.c

```diff
@@ -2163,6 +2163,19 @@ static int link_set_can(Link *link) {
                         return log_link_error_errno(link, r, "Could not append IFLA_CAN_RESTART_MS attribute: %m");
         }
 
+        if (link->network->can_triple_sampling > -1) {
+                struct can_ctrlmode cm = { .mask = CAN_CTRLMODE_3_SAMPLES, .flags = 0 };
```

Comment:
we usually would line break this like this:

```c
 struct can_ctrlmode cm = {
        .mask = CAN_CTRLMODE_3_SAMPLES,
 };
```

And given that `.flags` shall be zero we wouldn't bother with initializing it.